pkg/gadgets/trace/exec/tracer/gadget.go

```diff
@@ -23,7 +23,8 @@ import (
 )
 
 const (
-	ParamCwd = "cwd"
+	ParamCwd    = "cwd"
+	ParamFailed = "failed"
```

Comment:
We have a similar `IgnoreErrors` param in trace bind https://github.com/inspektor-gadget/inspektor-gadget/blob/9a0941298c639e4a52d1d98b9c69b1cb603f113a/pkg/gadgets/trace/bind/tracer/gadget.go#L28

Should we name both of them the same?

Reply:
IMO it's better to stick to the same pattern all around Inspektor Gadget. I'm ok with changing `ignore-errors` to `failed` in trace bind, but those should be aligned, otherwise it's difficult for users.

Regarding the default value, should we ignore failed operations by default? IMO they are relevant and could help to diagnose an issue, so I think we should show them by default. If they're creating too much noise, the user can disable them.